docs: Edge Cloud-init reference page (#1153)
spectrocloud/librarium: content/docs/04-clusters/03-edge/09-site-deployment/05-site-installation.md

Ideally, all Edge hosts have completed the *Install Handoff* phase when they arrive at the installation site. 

# Site User Data

You can provide a site-specific Edge Installer configuration user data if you need to apply new values or override default values from the Edge Installer user data that was created in the Installer Handoff phase. 
Follow the steps outlined in the [Build User Data ISO](/clusters/edge/site-deployment/prepare-edge-configuration#builduserdataiso) resource to create a site-specific user data. 

# Installation

Use the following steps to complete the Edge host installation.

<br />

<InfoBox>

The community resource, [Painting with Palette](https://www.paintingwithpalette.com/tutorials/) has a great Edge Native tutorial. 

</InfoBox>

<br />

<Tabs identifier="environment">

<Tabs.TabPane tab="Bare Metal" key="bare-metal">

## Prerequisites

- Access to Palette and the ability to register an Edge host.

> - Access to network information about the physical site, specifically the network virtual IP address (VIP).

Author: ```suggestion
- Access to network information about the physical site, specifically the network Virtual IP Address (VIP).
```

## Validation

1. Log in to [Palette](https://console.spectrocloud.com).

2. Navigate to the left **Main Menu** and select **Clusters**.

3. Select the host cluster you created to view its details page.

4. Review the **Cluster Status**. Ensure the **Cluster Status** field displays **Running**.

You can also use `kubectl` to issue commands against the cluster. Check out the [Access Cluster with CLI](/clusters/cluster-management/palette-webctl#overview) to learn how to use `kubectl` with a host cluster.

</Tabs.TabPane>

<Tabs.TabPane tab="VMware" key="vmware">

Use the following steps to complete the Edge host installation in a VMware environment.

## Prerequisites

- Access to Palette and the ability to register an Edge host.

- Access to network information about the physical site, specifically the network virtual IP address (VIP).

- Physical access to the Edge host.

- An Edge Installer OVF template. Check out the [Prepare Edge Hosts for Installation](/clusters/edge/site-deployment/stage) for guidance on how to create an Edge Installer OVF template.

## Site Install

> Perform the following steps to proceed with the installation at the site in your VMware environment.

Author: ```suggestion
Perform the following steps to proceed with the installation at the site in your VMware environment.

<br / >
```

1. Log in to vCenter Server by Using the vSphere Client.

2. Navigate to **VMs and Templates** and right-click on the desired folder and select the option to **Deploy VM(s) from this OVF template**.

3. Specify the location of the OVF template and start the deployment.

4. Proceed through the installation steps and deploy the virtual machine.

5. The VM will start up in the registration phase and wait for you to register the Edge host with Palette. If you provided the Edge Installer user data with an `EdgeHostToken` then the Edge host will automatically register with Palette. Otherwise, the Edge host will wait until you manually register the device in Palette. Go ahead and register the Edge host with Palette. Review the [Register Edge Host](/clusters/edge/site-deployment/site-installation/edge-host-registration) for additional guidance.

    <br />

    <InfoBox>

    Once the Edge host is registered, Palette will wait for you to create a host cluster and assign the  Edge host to the cluster.

    </InfoBox> 

> 6. The last step is to create a cluster definition if you don't have a host cluster for the Edge host to become a member of. Follow the steps in the [Create Cluster Definition](/clusters/edge/site-deployment/site-installation/cluster-deployment) to complete the site installation. 

Author: ```suggestion
6. The last step is to create a cluster if you don't have a host cluster that the Edge host can join. Follow the steps in the [Create Cluster Definition](/clusters/edge/site-deployment/site-installation/cluster-deployment) to complete the site installation. 
```

> Once the cluster is defined, the installation process will continue. The Palette Edge Host agent will start downloading all the required artifacts and reboot the Edge host. 

Author: ```suggestion
Once the cluster is created, the installation process will continue. The Palette Edge Host agent will download all required artifacts and reboot the Edge host. 
```
